src/fabric/test/fabric2_db_crud_tests.erl

```diff
@@ -250,6 +259,23 @@ scheduled_remove_deleted_dbs(_) ->
     end, DbNameList).
 
 
+scheduled_remove_deleted_dbs_with_error(_) ->
+    meck:expect(fabric2_db_expiration, process_expirations, fun(_, _) ->
+        throw(process_expirations_error)
+    end),
+
+    {Pid, Ref} = spawn_monitor(fun() ->
+        fabric2_db_expiration:cleanup(true)
+    end),
+    receive
+        {'DOWN', Ref, process, Pid, Error} ->
+            ?assertMatch({job_error, process_expirations_error, _}, Error)
+    end,
+    ?assertMatch({ok, _}, couch_jobs:get_job_data(
+        undefined, ?DB_EXPIRATION_JOB_TYPE, fabric2_db_expiration:job_id())
```

Comment:
We would want to perhaps also assert that the job is scheduled to run, let's just check get_job_state as well there.